lib/src/web/rtc_video_view_impl.dart

```diff
@@ -10,13 +10,16 @@ class RTCVideoView extends StatefulWidget {
     Key key,
     this.objectFit = RTCVideoViewObjectFit.RTCVideoViewObjectFitContain,
     this.mirror = false,
+    this.filterQuality = FilterQuality.low
```

Comment:
Let's add `,` at the end, to reduce diff when any additional parameter will be added.